mlflow/johnsnowlabs/__init__.py

```diff
@@ -787,6 +789,10 @@ def _get_or_create_sparksession(model_path=None):
     _validate_env_vars()
 
     spark = _get_active_spark_session()
+    if not MLFLOW_JOHNSNOWLABS_MODEL_REUSE_SPARK_SESSION.get():
```

Comment:
should this be `if not MLFLOW_JOHNSNOWLABS_MODEL_REUSE_SPARK_SESSION.get() and spark:`? Otherwise this causes an error for the case where there is no existing spark session: `MLFLOW_JOHNSNOWLABS_MODEL_REUSE_SPARK_SESSION = false` but `spark = None`